src/main/cli/settings.c

```diff
@@ -1227,6 +1227,7 @@ const clivalue_t valueTable[] = {
 #ifdef USE_RX_LINK_QUALITY_INFO
     { "osd_warn_link_quality",      VAR_UINT32  | MASTER_VALUE | MODE_BITSET, .config.bitpos = OSD_WARNING_LINK_QUALITY,     PG_OSD_CONFIG, offsetof(osdConfig_t, enabledWarnings)},
 #endif
+    { "osd_warn_over_cap",      VAR_UINT32  | MASTER_VALUE | MODE_BITSET, .config.bitpos = OSD_WARNING_OVER_CAP,     PG_OSD_CONFIG, offsetof(osdConfig_t, enabledWarnings)},
```

Comment:
Can you please add spaces here to align this entry with the ones above / below it?